example/linux/installation.md

```
@@ -0,0 +1,62 @@
+
+5. **View the Logs in Real-Time**:
+   sudo journalctl -u talawa-api.service -f
+
+6. **Stop the Talawa API Service:**:
+   sudo systemctl stop talawa-api.service
+
+7. **Restart the Talawa API Service:**:
+   sudo systemctl restart talawa-api.service
+
+8. **Also u can see direct log stored in /talawa-api/example/linux/talawa-api.log :**
+    cat pathOfLogFile
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)